modin/pandas/test/utils.py

```diff
@@ -21,7 +21,7 @@
     assert_index_equal,
     assert_extension_array_equal,
 )
-from modin.config.envvars import NPartitions
+from modin.config.envvars import MinElementsInPartition, NPartitions
```

Comment:
```suggestion
from modin.config import MinElementsInPartition, NPartitions
```
All objects from `envvars` are available from `modin.config`